src/Silhouette.js

```diff
@@ -27,6 +27,17 @@ const getPoint = ({_width: width, _height: height, _data: data}, x, y) => {
     return data[(y * width) + x];
 };
 
+const _default4b = new Uint8ClampedArray(4);
+
+const getColor4b = ({_width: width, _height: height, _colorData: data}, x, y) => {
```

Comment:
Pass a `dst` argument that defaults to a `new Uint8ClampedArray(4)` and set its indices manually. The default I think could be replaced with `.fill(0)`.